web/src/lib/components/admin-page/settings/template-settings/template-settings.svelte

```diff
@@ -0,0 +1,132 @@
+<script lang="ts">
```

Comment:
Since we are on Svelte 5, can you use the Rune syntax? https://svelte.dev/docs/svelte/what-are-runes